mullvad.js

```diff
@@ -18,16 +18,13 @@ const DEFAULT_ITEMS = {
     type: {name: _('VPN Type'), text: '', gSetting: 'show-type'},
 };
 
+const IO_ERROR = 7;
+
 const _networkMonitor = Gio.NetworkMonitor.get_default();
 
-const _httpSession = new Soup.SessionAsync();
-Soup.Session.prototype.add_feature.call(
-    _httpSession,
-    new Soup.ProxyResolverDefault()
```

Comment:
Is this no longer needed? I honestly forgot why I added this, but I remember at one point needing this.

Reply:
I don't think so. I can't tell what this used to do, but the extension seems to work without it.

Reply:
Yeah, it's entirely possible that whatever I used it to fix has been fixed upstream in newer versions of libsoup.